content/verification/veteran-verification/release-notes/2024-03-21.md

```diff
@@ -0,0 +1 @@
+We have updated the response for /disability ratings. Diagnostic_type_code under individual_ratings will now include all values associated with the disability rating from the VA source system. The lighthouse response concatenates values when more than one is provided for a disability rating, inserting a hypen character "-" between each value.
```

Comment:
Long story, but we're trying to not refer to ourselves as "Lighthouse" to public party consumers. Just an fyi! So "our" will work here instead.

```suggestion
We've updated the response for /disability ratings. Diagnostic_type_code under individual_ratings will now include all values associated with the disability rating from the VA source system. Our response concatenates values when more than one is provided for a disability rating, inserting a hypen character "-" between each value.
```